### PR TITLE
Bluetooth_Le_Audio: MICP

### PR DESCRIPTION
**Pull request by Meshram-Vibhor**
Microphone Input Control renamed to Microphone Control
Signed-off-by: Vibhor Meshram <vibhor69meshram@gmail.com>

### CODE REVIEW

**Review by Thalley — APPROVED** [review]

File: doc/releases/release-notes-2.7.rst [hunk 1]
```diff
@@ -249,7 +249,7 @@ Bluetooth
 
 * Audio
 
-  * Added the Microphone Input Control Service and client.
```
Comment (Thalley) [inline doc/releases/release-notes-2.7.rst:252]: Don't modify old release notes please :)

**Review by Thalley — CHANGES_REQUESTED** [review]

**Comment by Thalley** [comment]
@Meshram-Vibhor this also needs to be rebased to fix the conflict in uuid.h :)

File: doc/releases/release-notes-2.7.rst (hunk 1 above)
Comment (Meshram-Vibhor) [reply doc/releases/release-notes-2.7.rst:252]: Release notes kept unchanged.

Comment (Thalley) [reply doc/releases/release-notes-2.7.rst:252]: It is still changed from 
```
Added the Microphone Input Control Service and client.
```
to
```
Added the Microphone Input Control Service Service and client.
```
(note the extra "Service")

**Review by Thalley — CHANGES_REQUESTED** [review]
The release notes still needs to be fixed. Please also follow the contribution workflow defined by https://docs.zephyrproject.org/latest/contribute/guidelines.html#contribution-workflow (i.e. remove the merge commit and squash your duplicated commits).

**Comment by Thalley** [comment]
@Meshram-Vibhor Don't worry about the mess up :) It's not the first, nor will it be the last, time we see this happening :) Will you try to create a new PR, or fix this one?

**Comment by Meshram-Vibhor** [comment]
Apologies, I messed it up. I'll fix this and create a  new PR.

**Pull request closed without merging**